docs/build/developer-reference/l1-l2-interop.md

```diff
@@ -25,6 +25,14 @@ L1 to L2 communication is governed by the [`IZkSync.sol`](https://github.com/mat
 - If you prefer to learn-by-doing, the [cross chain governance tutorial](../tutorials/smart-contract-development/cross-chain-tutorial.md) is a practical example of layer interoperability.
   :::
 
+#### L1 to L2 transaction filtering
+
+Hyperchain operator can filter L1→L2 transactions coming through the Diamond proxy. It is achieved by having an address
+of the special contract called TransactionFilterer in the state transition storage. If the filterer exists, it is being
```

Comment:
You don't mean that it has to be called that, right? The only reqs are to set its address in the right place and have it implement the right interface, right?

Reply:
yes, that was a poor wording, fixed it